NewHorizons/Patches/ProxyBodyPatches.cs

```diff
@@ -46,5 +51,72 @@ public static void ProxyOrbiter_OnDestroy(ProxyOrbiter __instance)
             GlobalMessenger.RemoveListener("EnterMapView", __instance.OnEnterMapView);
             GlobalMessenger.RemoveListener("ExitMapView", __instance.OnExitMapView);
         }
+
+        [HarmonyReversePatch]
```

Comment:
do we need these patches? they look identical to the original methods

Reply:
To fix proxies when the moons are moved or sand funnel gets destroyed